assets/js/signup.js

```diff
@@ -23,7 +11,7 @@ var loadQuestions = function() {
   });
 
   $.getJSON({
-    url: '/server/form.php?retrieve=questions',
+    url: '/server/form.php?retrieve=questions&subheadings[]=Contact Information&subheadings[]=Language Information&subheadings[]=Background Information',
```

Comment:
Why do you have to specify the subheading[] for this? I'd really like for this to not be hard-coded, but I wanna hear the reasoning for this.